packages/cli/src/lib/Cli.ts

```diff
@@ -48,6 +50,10 @@ export class Cli {
     }
 
     switch (command) {
+      case 'updateConfig':
+        return await updateConfig(parsedArgs);
+      case 'doctor':
```

Comment:
You'll want to add these new commands to the help output (see line 58).

Reply:
I didn't understand exactly what did you mean, the user will need to type bubblewrap help doctor?

Reply:
If you run `bubblewrap help`, it prints out all the `bubblewrap` commands and tells you how to use them. Since you're adding new bubblewrap commands, you should add them to the output of `bubblewrap help`.